<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="lblTitle">
          <property name="text">
           <string>                                        SMART HOME</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtUserName">
          <property name="placeholderText">
           <string>User Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtPassword">
          <property name="echoMode">
           <enum>QLineEdit::EchoMode::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnAdmin">
          <property name="text">
           <string>ADMIN</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnUser">
          <property name="text">
           <string>USER</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2"/>
      <widget class="QWidget" name="page_3">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QListWidget" name="listWidgetRooms"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
